src/inlines/AutoLink.php

```diff
@@ -58,17 +58,22 @@ public static function consume(
 
     $uri = Str\slice($string, $start, $end - $start);
     if (\preg_match(self::ABSOLUTE_URI_PATTERN, $uri) === 1) {
-      return tuple(
-        new self($uri, $uri),
-        $offset,
-      );
+      if (!$context->areAllURISchemesEnabled()) {
+        $allowedURIs = $context->getAllowedURIs();
```

Comment:
This context method should still be renamed to be 'uri schemes', ditto for the variables.

Additionally, variables should be under_scored

Reply:
oof thank you for catching this